primitives/core/src/bounded/bounded_vec.rs

```diff
@@ -720,6 +730,24 @@ impl<T, S: Get<u32>> TruncateFrom<Vec<T>> for BoundedVec<T, S> {
 	}
 }
 
+impl<T, S: Get<u32>> Min<Vec<T>> for BoundedVec<T, S> {
```

Comment:
This is not how you would go about implementing this.

You need a blanket implementation along the lines of: 

```
impl<T, U> Min<T> for U {
  fn min(&self, t: T) -> Self { // compare self and t, return the smaller one } 
```